platform/solr/platform-solr-server-standalone/src/main/resources/solr/conf/solrconfig.xml

```diff
@@ -1140,6 +1140,38 @@
         </arr>
     </requestHandler>
 
+    <!-- Placename Suggestion Component
+
+    This component can return a list of auto-complete suggestions
+    when entering placenames.
+    -->
+    <searchComponent name="suggestPlace" class="solr.SuggestComponent">
```

Comment:
❗️ This search component should be named something more generic since it should support multiple dictionaries in the future.
https://lucene.apache.org/solr/guide/6_6/suggester.html#Suggester-MultipleDictionaries